src/test/java/org/jenkinsci/plugins/blueoceandisplayurl/BlueOceanDisplayURLImplTest.java

```diff
@@ -223,7 +251,6 @@ public static MultiBranchTestBuilder createProject(JenkinsRule j, String name, G
             return new MultiBranchTestBuilder(j, mp);
         }
 
-
```

Comment:
Please avoid making formatting changes to unrelated parts of a file. PR diffs should be as short as possible to accomplish the stated goal.